### ISSUE TITLE
Elements must only use allowed ARIA attributes for aria-label

### ISSUE THREAD

**Issue by jenniemc** [department-of-veterans-affairs/va.gov-team#24894]
## Summary
While it is not a critical or a serious issue, the use of aria-label came up as a potential issue through the Axe DevTools scan for the detailed appointment page within the Homepage Refresh flow. It marked the state abbreviation with "Ensure ARIA attributes are allowed for an element's role" 

The intent of the using "aria-label" is to supplement the state abbreviation for fully spelled out states for the screen readers.  The element where aria-label is currently used however, does not fall under the qualified uses based on https://www.aditus.io/aria/aria-label/:

WHAT ELEMENTS CAN BE USED WITH ARIA-LABEL

- **interactive elements**: such as a (when href attribute is present), audio and video (when controls attribute is present), input, select, button, textarea.
- **implicit landmark elements**: such as header, footer, nav, main, aside,section, and form.
- **explicit landmark elements**: such as an element with role="navigation".
- **widget role elements**: such as an element with role="dialog" or role="tooltip"(there are 27 widget roles in ARIA 1.1).
- **iframe and img elements**.

## Specs
- Device: MAC
- Browser:  Chrome
- Finding from: #24020

## Steps to Reproduce
**Test User:** 

1. Go to: health-care/schedule-view-va-appointments/appointments/
2. Click on any appointment card list that's Cheyenne VA Medical Center
3. Run the axe DevTool scan
4. 

### Result from Axe scan
![image.png](https://images.zenhubusercontent.com/5d6463ed1325bafe0674800b/0fb759a6-12ea-4342-a899-9e6ea183a1d4)

![image.png](https://images.zenhubusercontent.com/5d6463ed1325bafe0674800b/657ed835-959f-4360-a663-a5e552c759dc)

### Desired Result
screen reader reads the fully spelled out states and skips reading the abbreviation. Passes the Axe scan 

## Definition of Done
- [ ] screen reader reads the fully spelled out states and skips reading the abbreviation
- [ ] Passes the Axe scan

---
## How to configure this issue
- [ ] **Labeled with Team** (`vaos`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `design`, `product`)
- [ ] **Labeled with** `Bug`
- [ ] **Assigned to** Product for prioritization

**Comment by jbalboni** [comment]
I believe the markup we're using was provided by @1Copenut, so I don't think this is something we need to change, unless VSP accessibility folks thing we should.

**Comment by laurenernest** [comment]
@1Copenut @noahgelman we were asked to use aria-label in #18678. should we ignore this flag from axe?

**Comment by 1Copenut** [comment]
@laurenernest I took another look at this issue, and found a slightly improved way to announce the full state name, without throwing an axe warning:

```html
<span role="group">
  <span class="sr-only">Wyoming</span>
  <span aria-hidden="true">WY</span>
</span>
```

Can we adjust the HTML structure to match this snippet?

**Comment by laurenernest** [comment]
@jbalboni @jenniemc any concerns with the suggested HTML update?

**Comment by jbalboni** [comment]
@laurenernest Nope, it looks good to me.

**Comment by laurenernest** [comment]
Moving to dev

**Comment by laurenernest** [comment]
Submitted to VSP for review https://dsva.slack.com/archives/CBU0KDSB1/p1624549127273300

**Comment by laurenernest** [comment]
VSP will validate when we submit Homepage Refresh for the full 508 Audit. closing ticket.
